website/public/.htaccess

```diff
@@ -1,8 +1,15 @@
 # Copyright (C) 2020  Matthew "strager" Glazar
 # See end of file for extended copyright information.
 
-# Optimize loads of the main page and press release pages using HTTP/2 push.
 <IfModule http2_module>
+  Header always set Strict-Transport-Security "max-age=63072000; includeSubDomains"
+  Header always set X-Frame-Options "SAMEORIGIN"
+  Header always set Referrer-Policy "same-origin"
+  Header set X-Content-Type-Options "nosniff"
+  Header add Content-Security-Policy "default-src 'self';"
```

Comment:
Do we need `'unsafe-inline'` for `default-src`? I think we have some inline scripts, e.g. for some blog posts.